### PR TITLE
Initial release of item package selection

### PR DESCRIPTION
**Pull request by Lemongrass3110**
* **Addressed Issue(s)**: None

* **Server Mode**: Renewal

* **Description of Pull Request**: 
Adds support for the item package selection.

### CODE REVIEW

**Comment by HaRo22092022** [comment]
ohhh wow 💯 i like you Lemon but its work without 2022 exe?

**Comment by Lemongrass3110** [comment]
> ohhh wow 💯 i like you Lemon but its work without 2022 exe?

@HaRo22092022 No. You need a 2022 client.

**Comment by Pokye** [comment]
Wow. what a fantastic release. thank you Lemongrass <3

**Comment by bastardclaymore** [comment]
thank you
but where can i get the exe 2022-03 or above
Nemo newest released 2022-01-19 for now

**Comment by HaRo22092022** [comment]
> hank you
> but where can i get the exe 2022-03 or above
> Nemo newest released 2022-01-19 for now

buy from 4144 on nemo dc for 400$ 
but
Payment methods ONLY: webmoney, bitcoin (BTC), litecoin (LTC)

**Comment by JohnnyPlayy** [comment]
Another amazing work by this great worker and code artist. ❤️‍🔥⚡

File: db/import-tmpl/item_packages.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+###########################################################################
+# Item Package Database
+###########################################################################
+#
+# Item Package Settings
+#
+###########################################################################
+# - Item                       Item that triggers Item Package Selection.
+#   Groups:                    Groups that can be selected.
+#     - Group                  Group ID that can be selected.
```
Comment (Atemo) [inline db/import-tmpl/item_packages.yml:27]: We could leave a note that the ID must be the same as the client's BoxIndex (if I understand correctly)

File: db/re/item_packages.yml [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+        Items:
+          - Item: C_Classical_Fhat
+      - Group: 8
+        Items:
+          - Item: Wing_Of_Fly
+            Amount: 3
+            Rentalhours: 24
+          - Item: Wing_Of_Butterfly
+            Amount: 5
+            Rentalhours: 168
```
Comment (aleos89) [inline db/re/item_packages.yml:157]:
```suggestion
            RentalHours: 168
```

File: db/re/item_packages.yml [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+          - Item: Old_Blue_Box
+            Rentalhours: 1
```
Comment (aleos89) [inline db/re/item_packages.yml:159]:
```suggestion
            RentalHours: 1
```

**Review by aleos89 — APPROVED** [review]
LGTM!

**Comment by OptimusM** [comment]
First of all, thx for this amazing release as always.
I think it could be good if this UI could block the usage of cart, npc, storage and so on.

**Pull request merged**